class.json-api-endpoints.php

```diff
@@ -96,12 +96,14 @@
 
 	/**
 	 * @var bool Set to true if the endpoint accepts all cross origin requests
-	 *    You probably should not set this flag. If you are thinking of setting it,
-	 *    then discuss it with someone:
-	 *       http://operationapi.wordpress.com/2014/06/25/patch-allowing-endpoints-to-do-cross-origin-requests/
```

Comment:
Or at least hide it in Jetpack. On the wpcom version, you can use the commenting style mentioned in the "Adding Files to the Build Script" article in the FG to selectively hide comments from JP that would remain on wp.com